server/sonar-web/src/main/js/components/common/CodeSnippet.tsx

```diff
@@ -17,37 +17,31 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-import React from 'react';
-import classNames from 'classnames';
-import ClipboardButton from '../../../../components/controls/ClipboardButton';
-import { translate } from '../../../../helpers/l10n';
+import * as React from 'react';
+import * as classNames from 'classnames';
+import ClipboardButton from '../controls/ClipboardButton';
+import './CodeSnippet.css';
 
-/*::
-type Props = {
-  command: string | Array<?string>,
-  isOneLine?: boolean
-};
-*/
+interface Props {
+  className?: string;
+  isOneLine?: boolean;
+  noCopy?: boolean;
+  snippet: string | (string | undefined)[];
+}
 
 // keep this "useless" concatentation for the readability reason
 // eslint-disable-next-line no-useless-concat
 const s = ' \\' + '\n  ';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a multiline template literal instead of concatenating string literals with newlines. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWGfGEVFjNA6aRBI4UCP&open=AWGfGEVFjNA6aRBI4UCP)

[ISSUE_KEY]: AWGfGEVFjNA6aRBI4UCP